### ISSUE TITLE
Type errors

### ISSUE THREAD

**Issue by jordanpurinton** [Constructor-io/constructorio-ui-autocomplete#116]
Version:
```json
"@constructor-io/constructorio-ui-autocomplete": "^1.16.0",
```

Getting the following on the latest version:

> Could not find a declaration file for module '@constructor-io/constructorio-ui-autocomplete'. '/path/node_modules/@constructor-io/constructorio-ui-autocomplete/lib/mjs/index.js' implicitly has an 'any' type.
>   There are types at '/path/node_modules/@constructor-io/constructorio-ui-autocomplete/lib/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@constructor-io/constructorio-ui-autocomplete' library may need to update its package.json or typings.

**Comment by esezen** [comment]
Hello @jordanpurinton , we are looking into this issue and we will get you an update as soon as possible. 
Would you be able to provide the following?

- How are you importing the library?
- What version of Next are you on?
- What version of Node are you on?

**Comment by jordanpurinton** [comment]
I'm using your example for `useCioAutocomplete` and importing it like so:

```ts
import { useCioAutocomplete } from "@constructor-io/constructorio-ui-autocomplete";
```

- **Next**: `14.0.4`
- **Node**: `18.17.0`

**Comment by jordanpurinton** [comment]
- `npx create-next-app@latest`
- `cd your-app`
- `npm i @constructor-io/constructorio-ui-autocomplete`
- Create a component and paste in the snippet from the `useCioAutocomplete` documentation
- Observe error in ticket in import statement for `useCioAutocomplete`

**Comment by jordanpurinton** [comment]
Here are the deps from that newly generated project:
```json
  "dependencies": {
    "@constructor-io/constructorio-ui-autocomplete": "^1.16.0",
    "next": "14.0.4",
    "react": "^18",
    "react-dom": "^18"
  },
  "devDependencies": {
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "typescript": "^5"
  }
```

**Comment by mocca102** [comment]
Hey @jordanpurinton issue resolved here in [v1.16.1](https://github.com/Constructor-io/constructorio-ui-autocomplete/releases/tag/v1.16.1)

**Comment by jordanpurinton** [comment]
Resolved on my end, thanks! 👍
